src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -180,10 +181,17 @@ internal void RaiseSizeChanged(SizeChangedEventArgs args)
 		{
 #if !__NETSTD_REFERENCE__ && !IS_UNIT_TESTS
 			SizeChanged?.Invoke(this, args);
+#if !UNO_HAS_ENHANCED_LIFECYCLE
 			_renderTransform?.UpdateSize(args.NewSize);
+#endif
 #endif
 		}
 
+#if UNO_HAS_ENHANCED_LIFECYCLE
+		internal void UpdateRenderTransformSize(Size newSize)
+			=> _renderTransform?.UpdateSize(newSize);
```

Comment:
FYI I think this is actually useless on skia (and it should be) ... we can keep it if we want to keep the contract of `NativeRenderTransformAdapter` unified between platforms, but I think it would make sense to update it also.

Reply:
Why is it useless on Skia?

Reply:
Well I was assuming that all `Transform` info were pushed to the `Visual` and they will be applied during the painting / rendering using the size of the `Visual` itself. But actually looking at the implementation of the `NativeRenderTransformAdapter`(https://github.com/unoplatform/uno/blob/master/src/Uno.UI/Media/NativeRenderTransformAdapter.skia.cs#L28C60-L28C68) we are computing the matrix using the `CurrentSize`... which is probably wrong but not the subject of this PR!